pkg/cmd/kafka/update/update.go

```diff
@@ -281,20 +282,23 @@ func promptConfirmUpdate(opts *options) (bool, error) {
 // creates a summary of what values will be changed in this update
 // returns a formatted string. Example:
 // owner: foo_user	➡️	bar_user
+// to do: Needs update once reauthentication_enabled is incorporated
 func generateUpdateSummary(new reflect.Value, current reflect.Value) string {
 	var summary string
 
 	newT := new.Type()
 
-	for i := 0; i < new.NumField(); i++ {
+	for i := 0; i < 1; i++ {
```

Comment:
I'm just curious what this does, how it fixes it?

Reply:
This is to hide the `reauthentication_nabled` from preview.

Reply:
Yes I understand _why_ it was done, I just did not yet understand _how_. I've debugged locally to understand it now.

This is a "quick fix" but once reauth update is possible we won't be able to just revert this change. If the update does not include and change to `reauthentication_enabled`, reverting this will still include it in the update summary.

I see that the value of ReauthenticationEnabled is nil, so perhaps we can check for nil before including it in the update summary?

This also assumes that the owner field will always be the first property of the update object, we should not rely on this.